src/kinvey.d.ts

```diff
@@ -1,11 +1,532 @@
 // Type definitions for Kinvey JavaScript SDK
 // Definitions by: Kinvey <https://github.com/Kinvey>
 
-import { Observable } from 'rxjs/Observable';
+import {
+  Observable
+} from 'rxjs/Observable';
 
-export as namespace Kinvey;
+export namespace Kinvey {
```

Comment:
Clearly this will work, but it involves more maintaining since we now have 2 copies of exports in the same file. I take that there's no easier way to keep backwards compatibility. 

A couple of follow-ups that'll help:
- Add a comment above this export, briefly describing why it exists and suggesting that it is deprecated. It helps us as well as SDK users who look at our typescript definition.
- If we are recommending the "new" way of importing a definition, a devcenter PR that accompanies this release (or follows soon after) should change the snippets.